compile.bat

```diff
@@ -0,0 +1,13 @@
+@echo off
+SET /p site="Enter site (leuven, antwerpen, gent, brussel): "
+SET /p os="Enter OS (windows, linux, mac): "
+SET /p document="Enter document (intro-Linux, intro-HPC, perfexpert): "
```

Comment:
Not sure if it's worth the trouble, but do we need to check whether the answer makes sense, i.e. take into account typos?